bodhi/server/templates/master.html

```diff
@@ -24,8 +24,10 @@
     % else:
         <title>${update.title} - Fedora Updates System</title>
     % endif
-    <link href="${request.static_url('bodhi:server/static/bootstrap/css/bootstrap.css') }" rel="stylesheet" />
-    <link href="${request.static_url('bodhi:server/static/font-awesome-4.4.0/css/font-awesome.min.css') }" rel="stylesheet" />
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-1.0.1/fedora-bootstrap.css" rel="stylesheet" type='text/css' />
+    <link href='https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css' rel='stylesheet' type='text/css' />
+    <link href='https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/hack.css' rel='stylesheet' type='text/css' />
+    <link href='https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/font-awesome.css' rel='stylesheet' type='text/css' />
```

Comment:
Fedora is actually not the only user of Bodhi. I've had a couple of pull requests from other users of Bodhi that had their own deployments. Due to the CORS issue you cited, and also due to us not knowing if all Bodhi deployments are on the public internet I don't think we should rely on this way of hosting these web assets.

How about we just bundle these assets into Bodhi for now, since that's how Bodhi has been doing this up until now? I can look into unbundling it later - even though bundling isn't ideal it's not a step backwards since Bodhi is alread doing it.